http3/roundtrip.go

```diff
@@ -69,7 +69,7 @@ type RoundTripper struct {
 
 	// Dial specifies an optional dial function for creating QUIC
 	// connections for requests.
-	// If Dial is nil, quic.DialAddrEarlyContext will be used.
+	// If Dial is nil, makeDialer will be used to create a dialer.
```

Comment:
That's not really a helpful comment for the user of this package. Can we describe instead what happens (a single packet conn is reused)?

Reply:
How about this:
```
If Dial is nil, a single packet conn will be reused for dialer.
```